DonjinKrawler-Client/src/main/java/donjinkrawler/items/HealthPotion.java

```diff
@@ -19,8 +23,17 @@ public String getDescription() {
         return "Restore some HP.";
     }
 
+    @Override
+    public void accept(ItemVisitor v) {
```

Comment:
From the [checklist](https://dzone.com/articles/java-code-review-checklist):
`Use intention revealing names.`